.github/workflows/ci.yml

```diff
@@ -53,7 +52,7 @@ jobs:
         with:
           node-version: ${{matrix.node}}.x
 
-      - run: corepack yarn install --immutable
+      - run: corepack yarn install
```

Comment:
Is this necessary if the previous job has already made the necessary verification?
```suggestion
```

Reply:
Ah gotcha I thought we were still on zero-install.